eng/versioning/version_client.txt

```diff
@@ -115,7 +115,7 @@ com.azure:azure-perf-test-parent;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-quantum-jobs;1.0.0-beta.1;1.0.0-beta.2
 com.azure:azure-search-documents;11.4.5;11.5.0-beta.6
 com.azure:azure-search-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-security-attestation;1.0.0-beta.1;1.0.0-beta.2
+com.azure:azure-security-attestation;1.0.0-beta.1;1.0.0
```

Comment:
Did you do this manually? You may have to run the prepare-release script for this.

Reply:
For example in this scenario you would ideally want to change both the dependency and the current version.